ubcpi/static/css/ubcpi.css

```diff
@@ -0,0 +1,495 @@
+  border-radius: 2px;
+  margin-top: 20px;
+}
+
+.your-answer-and-rationale textarea {
+  min-height: 200px;
+  border: 1px solid #b9c1c8;
+  color: #3c3c3c;
+  overflow-y: scroll;
+}
+
+.results-container {
+  -webkit-transition: all 0.5s ease-in-out;
+  -moz-transition: all 0.5s ease-in-out;
+  transition: all 0.5s ease-in-out;
+  -webkit-transform: scale(1);
+  -moz-transform: scale(1);
+  -ms-transform: scale(1);
+  -o-transform: scale(1);
+  transform: scale(1);
+  -webkit-border-radius: 2px;
+  -moz-border-radius: 2px;
+  -ms-border-radius: 2px;
+  -o-border-radius: 2px;
+  border-radius: 2px;
+  box-shadow: 0 1px 1px 0 rgba(2,2,3,0.1);
+  margin-bottom: 20px;
+  border: 1px solid #d7dbdf;
+  border-top: 5px solid #d7dbdf;
+  padding: 10px 20px;
+  background: #fff;
+}
+
+.results-container h3 {
+  color: #65BBE4;
+  font-weight: 900;
+  border-bottom: 1px solid #D7DBDF;
+  padding-bottom: 10px;
+  margin-bottom: 20px;
+  font-size: 18px;
+}
+
+.wrapper-comp-settings .list-input.settings-list .add-image-button {
+
+}
+
+.wrapper-comp-settings .list-input.settings-list .add-image-button:disabled {
+
+}
+
+/* Graph Styling */
+.ubcpibar {
+    fill: #33b2eb;
+    opacity: 0.8;
+}
+
+.ubcpibar.correct-answer {
+    fill: #50c67b;
+}
+
+.ubcpibar:hover {
+    opacity: 1.0;
+}
+
+.correct-answer-option{
+    margin-bottom: 20px;
+    color: #50c67b;
+}
+
+#pi-form .list-input.settings-list .setting-label {
+    vertical-align: top;
+    margin-bottom: 20px;
+}
+
+#pi-form .list-input.settings-list .ubcpi-options-list-container .setting-label {
+    width: 47%;
+}
+
+#pi-form .list-input.settings-list .ubcpi-options-list-container .ubcpi-field {
+    width: 45%;
+    display: inline-block;
+}
+
+#pi-form .list-input.settings-list .setting-label + input,
+#pi-form .list-input.settings-list .setting-label + textarea,
+#pi-form .list-input.settings-list .setting-label + select {
+    margin-top: 20px;
+}
+
+.wrapper-comp-settings .list-input.settings-list .setting-help {
+    min-width: 100px;
+    margin-right: 10px;
+    color: rgb(140,140,140);
+    font-style: italic;
+    font-weight: normal;
+}
+
+.ubcpi-options-list-container {
+    width: 73%;
+    display: inline-block;
+}
+
+#pi-form .wrapper-comp-settings .list-input.settings-list .wrapper-comp-setting {
+    margin-bottom: 15px;
+}
+
+.option-details-text {
+    font-style: italic;
+    font-size: 0.8em;
+    opacity: 0.7;
+    clear: both;
+}
+
+.warning-notice {
+    display: block;
+    background: #edbd3c;
+    padding: 10px;
+    color: black;
+    margin: 40px 0 10px;
+}
+
+    div.course-wrapper section.course-content .warning-notice p {
+        margin-bottom: 0;
+        color: black;
+    }
+
+.correct-option {
+    color: #50c67b;
+}
+
+.possible-option {
+    font-style: italic;
+}
+
+.response-text {
+    padding-left: 22px;
+}
+
+/* The LMS view has different styling for pre tags */
+.ubcpipre {
+    font-family: 'Open Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
```

Comment:
why do you want to use Open Sans for pre tags? These tags generally represent code blocks, no?